### ISSUE TITLE
reduce dependence on external data servers

### ISSUE THREAD

**Issue by juliasloan25** [CliMA/ClimaLand.jl#625]
## Describe the bug
We download some of our data directly from external servers every time we run a simulation. This causes problems, e.g. when those servers are down we can't download data and can't run our simulations. We can address this by fully switching to use ClimaArtifacts.

## SNOTEL data bug description
Suddenly I get a docs failure on two separate branches that don't make any changes in `docs/`. It comes from `docs/tutorials/standalone/Snow/data_tutorial.jl` and seems to be due to a dimension mismatch when creating a dataframe.

```
ERROR: LoadError: LoadError: DimensionMismatch: Length of nms doesn't match length of x.
in expression starting at /home/runner/work/ClimaLand.jl/ClimaLand.jl/docs/src/generated/standalone/Snow/data_tutorial.md:2
when executing the following code block from inputfile `~/work/ClimaLand.jl/ClimaLand.jl/docs/tutorials/standalone/Snow/data_tutorial.jl`

metadata = DataTools.snotel_metadata();
metacols = ["id", "state", "elev", "lat", "lon"]
DataFrames.rename!(metadata, Symbol.(metacols));
```

Failing Github actions:
- [on js/regridder_type](https://github.com/CliMA/ClimaLand.jl/actions/runs/9199103878/job/25303238506)
- [on js/imp-interface](https://github.com/CliMA/ClimaLand.jl/actions/runs/9197994538/job/25299697860)

## To Reproduce
Ideally a [minimal reproducible example](https://stackoverflow.com/help/minimal-reproducible-example):

```julia
# some code here
```

Or even better, add it as a unit test, and open pull request.

<details><summary>Project</summary>
<p>
If not using the `examples` project:
```
paste your Project.toml here.
```
```
paste your Manifest.toml here.
```

</p>
</details>

## System details

Any relevant system information:
- Julia version
- operating system
- modules loaded on cluster (`module list`)

## Related issues / PRs

Please add any relevant links.

**Comment by Sbozzolo** [comment]
There were probably server issues. I meged the PR.

I think that the real issue is that we should try to avoid depending on an external server if the data we use is always the same.

**Comment by juliasloan25** [comment]
> There were probably server issues. I meged the PR.
> 
> I think that the real issue is that we should try to avoid depending on an external server if the data we use is always the same.

Agreed, I updated this issue to highlight that

**Comment by juliasloan25** [comment]
will be resolved as we move to ClimaArtifacts - see #580

**Comment by Sbozzolo** [comment]
My understanding is that part of the Snow code is supposed to grab and process whatever data the user from Snotel wants.

I also think that we had issues with the MODIS servers at some point. I am not sure where we access them.
